src/main/java/mate/academy/spring/dto/MovieSessionResponseDto.java

```diff
@@ -0,0 +1,42 @@
+package mate.academy.spring.dto;
+
+import java.time.LocalDateTime;
+
+public class MovieSessionResponseDto {
+    private Long id;
+    private Long movieId;
+    private Long cinemaHallId;
+    private LocalDateTime showTime;
```

Comment:
```suggestion
    private Long cinemaHallId;
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    private LocalDateTime showTime;
```
You can put a pattern here so that the Json field returns the date as a date, not as an array